udf/histogram_get_sum.sql

```diff
@@ -0,0 +1,18 @@
+/*
+
+Take a list of histograms of type STRUCT<key STRING, value STRING>,
+extract the histogram of the given key, and return the sum value
+
+ */
+CREATE TEMP FUNCTION udf_histogram_get_sum(histogram_list ANY TYPE, target_key STRING) AS (
+  (
+    SELECT
+      udf_json_extract_histogram(value).sum
+    FROM
+      UNNEST(histogram_list)
+    WHERE
+      key = target_key
+    LIMIT
+      1
+  )
+);
```

Comment:
@relud should we add tests for such simple udfs?

Reply:
IMO if it's complex enough to be a UDF, it's complex enough to need tests.